tests/serialization/testdata/serialize_by_file_shard/TestManifestSerializer/symlink_model_file

```diff
@@ -1 +0,0 @@
-.:0:22:3aab065c7181a173b5dd9e9d32a9f79923440b413be1e1ffcdba26a7365f719b
```

Comment:
all the other deleted tests are related to digest serializer. This is related to the manifest serializer?